### PR TITLE
updated remaining files with the lint errors

### PR DESCRIPTION
**Pull request by Rahul-Palamarthi**
Updated the code for the options so that they are available in mobile menu.

Also fixed the error showing in console for the footer component.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "packages": {
     "": {
       "name": "thecyberhub.org",
-      "version": "0.25.2",
+      "version": "0.25.3",
```
Comment (kabir0x23) [inline package-lock.json:9]:
```suggestion
      "version": "0.25.2",
```

File: package-lock.json [hunk 2]
```diff
@@ -12883,4 +12883,4 @@
       "dev": true
     }
   }
-}
\ No newline at end of file
+}
```
Comment (kabir0x23) [inline package-lock.json:12886]:
```suggestion
}

```

**Review by kabir0x23 — APPROVED** [review]
```
Great 😄

Thanks for contributing to this project.

If you want to contribute more to open-source then must check out other open-source projects and communities:
- [Thecyberworld](https://github.com/thecyberworld).
- [Cryptoverse Web3](https://github.com/CryptoverseWeb3)
- [kabir0x23](https://github.com/kabir0x23)

**Comment by Rahul-Palamarthi** [comment]
> ```
> Great 😄
> 
> Thanks for contributing to this project.
> 
> If you want to contribute more to open-source then must check out other open-source projects and communities:
> - [Thecyberworld](https://github.com/thecyberworld).
> - [Cryptoverse Web3](https://github.com/CryptoverseWeb3)
> - [kabir0x23](https://github.com/kabir0x23)
> ```

sure I will be looking these projects

**Pull request merged**